cpp/apidoc/tutorials/tensor_to_py.md

```diff
@@ -0,0 +1,128 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+Use Plasma to Access Tensor from C++ in Python
+==============================================
+
+This short tutorial shows how to use Arrow and Plasma Store to send data 
+from C++ to Python. 
+
+In detail, we will show how to:
+1. Serialize a floating-point array in C++ into Arrow Tensor
+2. Save the Arrow Tensor to Plasma In-Memory Object Store
+3. Access the Tensor in a Python Process
+
+This approach has the advantage that multiple python processes can all read 
+the tensor with zero-copy. Therefore, only one copy is necessary when we send
+a tensor from one C++ process to many python processes. 
+
+
+Step 0: Set up
+------
+We will include the following header files and construct a Plasma client.
+
+```cpp
+#include <plasma/client.h>
+#include <arrow/tensor.h>
+#include <arrow/array.h>
+#include <arrow/buffer.h>
+#include <arrow/io/memory.h>
+#include <arrow/ipc/writer.h>
+
+PlasmaClient client_;
+ARROW_CHECK_OK(client_.Connect("/tmp/plasma", "", 0));
+```
+
+
+Step 1: Serialize a floating point array in C++ into Arrow Tensor
+---------------------------------------------
+In this step, we will construct a floating-point array in C++. `generate_input`
+function takes the input length and returns the pointer to a `float` array 
+filled with random elements. 
+
+```cpp
+// Generate Random Object ID for Plasma, 20 bytes
+ObjectID object_id = ObjectID::from_random();
```

Comment:
Unfortunately it looks like the `ObjectID::from_random` API is being phased out. @pcmoritz

Reply:
Might be useful to restore it as a util method and ask that the user supply a seed